oauth2-authorization-server/src/main/java/org/springframework/security/config/annotation/web/configurers/oauth2/server/authorization/OAuth2AuthorizationServerConfigurer.java

```diff
@@ -144,6 +152,18 @@
 		return this;
 	}
 
+	/**
+	 * Sets the user info claims mapper.
+	 *
+	 * @param userInfoClaimsMapper the user info claims mapper
+	 * @return the {@link OAuth2AuthorizationServerConfigurer} for further configuration
+	 */
+	public OAuth2AuthorizationServerConfigurer<B> userInfoClaimsMapper(UserInfoClaimsMapper userInfoClaimsMapper) {
```

Comment:
I suggest changing this to accept `Function<Object, OidcUserInfo>`. See note on `UserInfoClaimsMapper`.